Smaller things, bugfixes & solving issue #102

kleeder opened the pull request:
added help for entry_id command,
fixed botb levelup array points,
added "points to next level" to levelup command,
made lyceum page search possible for titles smaller than 3 letters (56)
changed generic XHB to either OHB,2HB,4HB depending on hour count (Issue #102),
added function to determine hour-count of XHB,
added cohb/cxhb command to only get the currently running ohb (less spammy),
added debug-check to irc log endpoint,
added mathjs package and !calc command (Issue #79),
added major battle command and refactor ohb command (Issue #86),
added !format command (Issue #84)

Commit:
added help for entry_id command

Commit:
fixed botb levelup array points

Commit:
changed generic XHB to either OHB,2HB,4HB depending on hour count,
added function to determine hour-count of XHB,
added cohb/cxhb command to only get the currently running ohb (less spammy),
added debug-check to irc log endpoint

Commit:
added mathjs package and !calc command

Commit:
added major battle command and refactor ohb command

Commit:
added !format command

Commit:
added dami f. tune suggestion to search for format tokens via !format command

langel commented on README.md:
```diff
@@ -8,9 +8,9 @@ To install the prerequisites:
+You also need the LAME, SoX, curl and math.js packages. On Ubuntu/Debian, you can use the following command:
 
-``apt-get install lame sox curl``
+``apt-get install lame sox curl mathjs``
```
I'm not sure that adding mathjs here is necessary. Typically for a node package you would use npm (or yarn or similar) to install the node module locally to the project in `node_modules` using `npm install`.

langel commented on package.json:
```diff
@@ -5,7 +5,9 @@
-    "irc": "^0.5.0"
+    "irc": "^0.5.0",
+    "mathjs": "^13.0.1",
+    "sox": "^0.1.0"
```
Unless we actually start using sox inside of node we don't need this module. Right now we call it externally. This could be another future refactor though I'm not sure if its worth the time.

Commit:
removed install and dependency notes from package json and readme md after PR-review

kleeder commented:
oh right. i am not an expert when it comes to node-projects and their infrastructure/dependencies. i updated the PR and removed these changes from the files
